lib/dealership.rb

```diff
@@ -0,0 +1,43 @@
+class Dealership
+  attr_reader :name, :address, :inventory, :inventory_count
+
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+  end
+
+  def has_inventory?
+    if @inventory.count <= 0
+      false
+    elsif @inventory.count > 0
+      true
+    end
+  end
+
+  def inventory_count
+    @inventory.count
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def cars_by_make(make)
+    make_cars = []
+    @cars.each do |car|
```

Comment:
All of your cars are being stored in `@inventory` so if you change @cars to @inventory this would work!
```suggestion
    @inventory.each do |car|
```